src/nls/cs/strings.js

```diff
@@ -410,9 +422,11 @@ define({
     "CMD_TWITTER"                         : "{TWITTER_NAME} - Twitter",
     "CMD_ABOUT"                           : "O aplikaci {APP_TITLE}",
     "CMD_OPEN_PREFERENCES"                : "Otevřít soubor s preferencemi",
+    "CMD_OPEN_KEYMAP"                     : "Otevřít uživatelské klávesové zkratky",
 
     // Řetězce pro main-view.html
     "EXPERIMENTAL_BUILD"                   : "experimentální verze",
+    "RELEASE_BUILD"                        : "verze",
```

Comment:
I haven't known about this process before. Is there any plan to check it also with a grammar check tool? In my opinion it would be maybe better than Google Translator. Because as you can see in https://github.com/dvorapa/brackets/commit/692fb164c937595623c24794b170611d010c2297 commit from #10563 pull request, there were some typos and gramatical mistakes in the translation.